inc/gpio.h

```diff
@@ -19,42 +19,45 @@ typedef struct GPIOAddress {
 
 // For setting the direction of the pin
 typedef enum {
-  PIN_DIR_IN = 0,
-  PIN_DIR_OUT,
-} PinDir;
+  GPIO_DIR_IN = 0,
+  GPIO_DIR_OUT,
+} GPIODir;
 
 // For setting the output value of the pin
 typedef enum {
-  PIN_STATE_LOW = 0,
-  PIN_STATE_HIGH,
-} PinState;
+  GPIO_STATE_LOW = 0,
+  GPIO_STATE_HIGH,
+} GPIOState;
 
 // For setting the internal pull-up/pull-down resistor (Turn off if a hardware
 // PUPDR is enabled). Different on MSP and STM for MSP RES_UP and RES_DOWN both
-// set PIN_REN to 1. Could make this RES_ON, RES_OFF and make pullup and
-// pulldown vary with the PIN_VAL the same way it works for the MSP430
+// set GPIO_REN to 1. Could make this RES_ON, RES_OFF and make pullup and
```

Comment:
You should probably remove the device-specific portion of this comment before merging